xbmc/GUIInfoManager.cpp

```diff
@@ -10950,12 +10959,21 @@ bool CGUIInfoManager::ConditionsChangedValues(const std::map<INFO::InfoPtr, bool
 
 bool CGUIInfoManager::IsPlayerChannelPreviewActive() const
 {
-  bool streamValid = m_videoInfo.valid;
-  if (m_currentFile->HasPVRChannelInfoTag() && m_currentFile->GetPVRChannelInfoTag()->IsRadio())
-    streamValid = m_audioInfo.valid;
-
-  return m_playerShowInfo &&
-         (m_isPvrChannelPreview || !streamValid);
+  bool bReturn(false);
+  if (m_playerShowInfo)
```

Comment:
@ksooo I think we need to check for m_currentFile->HasPVRChannelInfoTag() at this level too